tests/src/Traits/ApigeeMonetizationTestTrait.php

```diff
@@ -144,7 +144,7 @@ protected function createAccount(array $permissions = [], bool $status = TRUE, s
       'first_name' => $this->randomMachineName(),
       'last_name' => $this->randomMachineName(),
       'name' => $this->randomMachineName(),
-      'pass' => user_password(),
+      'pass' => \user_password(),
```

Comment:
Is there a reason for a slash at the beginning of the function ?

Reply:
When the user module isn't enabled yet, php is looking for `Drupal\Tests\apigee_m10n\Traits\user_password()`.